.github/workflows/main.yml

```diff
@@ -12,16 +12,14 @@ jobs:
         with:
           persist-credentials: false
 
-      - name: Install #This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
-        uses: borales/actions-yarn@v2.3.0
-        with:
-          cmd: install # will run `yarn install` command
+      - name: Install
+        run: yarn install
+
       - name: Build
-        uses: borales/actions-yarn@v2.3.0
-        with: # will run `yarn build` command
-          cmd: |
-            build
-            cp ./config/CNAME ./build/CNAME
+        run: |
+          yarn build
+          cp ./config/CNAME ./build/CNAME
```

Comment:
Here I am unsure if `CNAME` was a variable from the yarn action, but I think it is very unlikely that this is the case. I guess you know that @brenzi?

Reply:
CNAME must be set correctly for gh pages to work on a custom DNS